nexus/db-queries/src/db/datastore/deployment.rs

```diff
@@ -3312,7 +3321,7 @@ mod tests {
         );
         assert_eq!(blueprint1.sleds.len(), collection.sled_agents.len());
         assert_eq!(
-            blueprint1.all_omicron_zones(BlueprintZoneDisposition::any).count(),
+            blueprint1.in_service_zones().count(),
```

Comment:
Just noting that this is no longer functionally the same, but I guess it happens to be right here.